<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsWindow</class>
 <widget class="QDialog" name="OptionsWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="buttonLogout">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Kirjaudu Ulos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonWithdraw">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>360</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Nosto</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonBalance">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>400</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Saldo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonTransactions">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>440</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Tapahtumat</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonChangeAccount">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Vaihda tili√§</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAccountName">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>641</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>tili: x</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
